modules/touch/src/directinputsolver.cpp

```diff
@@ -212,6 +212,13 @@ bool DirectInputSolver::solve(const std::vector<TouchInputHolder>& list,
                               const std::vector<SelectedBody>& selectedBodies,
                               std::vector<double>* parameters, const Camera& camera)
 {
+    ZoneScopedN("Direct touch input solver")
+
+    ghoul_assert(
+        selectedBodies.size() >= list.size(),
```

Comment:
This name is a little confusing. As I understand, these are selected points on a surface or body, rather than multiple selected nodes. Is that correct?

Reply:
I agree that it's confusing, but limited myself when it came to poking around in the code for the directinputsolver. As seen in the function signature, the struct for that varaible is currently also named "SelectedBody". I have changed the name in the touchInteraction class in another commit, so that it's a bit more obvious in the code that calls this function at least. 

But yes the `SelectedBody` is a struct with a touch input id, a pointer to a scene graph node and a surface position (or rather, a position in local coordinates on the interaction sphere). If you've got a good name suggestion, I could go ahead and rename it in this PR as well.

Reply:
I agree with limiting the refactoring and keeping away from the directinputsolver code. The slight confusion about the name is probably not worth it.